test/Microsoft.VisualStudio.Threading.Tests/TestBase.cs

```diff
@@ -276,7 +276,9 @@ protected void ExecuteOnSTA(Action action)
                 staFailure = ex;
             }
         });
+#pragma warning disable CA1416 // Validate platform compatibility
```

Comment:
💭 you would have the option of targeting `net5.0-windows` instead of just `net5.0`

Reply:
But I want most of these tests to run on other OS's, and all these tests already check the OS first, but the analyzer can't see that.